clients/ITAC_WinClient/MainWindow.xaml.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.TextFormatting;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using ITAC_WinClient.Models;
+
+
+namespace ITAC_WinClient
+{
+    /// <summary>
+    /// Логика взаимодействия для MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        
+        private string MsgDesc = "Server connected.";
+        public bool isConnected { get; set; }
+        public bool isAutorizated { get; set; }
+        public ConfigurationStruct configurationStruct;
+
+        public MainWindow()
+        {
+            isConnected = true;
+            isAutorizated = false;
+
+            if (!isConnected)
+                MsgDesc = "Server not found.";
+
+            configurationStruct.host = "127.0.0.1";
+            configurationStruct.port = "8089";
+
+           
+            InitializeComponent();
+        }
+
+        private void MenuItem_Click(object sender, RoutedEventArgs e)
+        {
+            // TODO: Сохранение данных
+            // Разрыв соединения с вервером
+            Close(); // Window close
+        }
+
+        private void MenuItem_Click_1(object sender, RoutedEventArgs e)
+        {
+            MessageBox.Show("ITAC - Universal client for ITAC messenger.\nVersion 14.6 (beta).\n" +
+                            "\nDevelopers:\n"+
+                            "1. Igor Moiseev (Product meneger).\n"+
```

Comment:
Who is meneger? May be you want to write 'manager'?

Reply:
yes)) thanks